docs/pages/foundations/icons.tsx

```diff
@@ -1,114 +0,0 @@
-import { useState } from 'react';
```

Comment:
This page was created because it has a mixture of MDX content + the icons playground.

But having this as it's own page is not ideal because it's exactly the same as `docs/pages/foundations/[slug].tsx`.

To fix this, I have moved the icons playground into the mdx file which removes the need for this file.